<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="119" started-at="2017-05-12T05:44:28Z" finished-at="2017-05-12T05:44:28Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="119" started-at="2017-05-12T05:44:28Z" finished-at="2017-05-12T05:44:28Z">
      <class name="script.Demo">
        <test-method status="FAIL" signature="testA()[pri:0, instance:script.Demo@7921b0a2]" name="testA" duration-ms="40" started-at="2017-05-12T11:14:28Z" finished-at="2017-05-12T11:14:28Z">
          <exception class="java.net.MalformedURLException">
            <message>
              <![CDATA[For input string: "204:5004:c06e:69ea:c64a:a6d7:6567:4444"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.net.MalformedURLException: For input string: "204:5004:c06e:69ea:c64a:a6d7:6567:4444"
	at java.net.URL.<init>(Unknown Source)
	at java.net.URL.<init>(Unknown Source)
	at java.net.URL.<init>(Unknown Source)
	at script.Demo.testA(Demo.java:21)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:823)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:778)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1225)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1150)
	at org.testng.TestNG.runSuites(TestNG.java:1075)
	at org.testng.TestNG.run(TestNG.java:1047)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
Caused by: java.lang.NumberFormatException: For input string: "204:5004:c06e:69ea:c64a:a6d7:6567:4444"
	at java.lang.NumberFormatException.forInputString(Unknown Source)
	at java.lang.Integer.parseInt(Unknown Source)
	at java.lang.Integer.parseInt(Unknown Source)
	at java.net.URLStreamHandler.parseURL(Unknown Source)
	... 29 more
]]>
            </full-stacktrace>
          </exception> <!-- java.net.MalformedURLException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testA -->
      </class> <!-- script.Demo -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
